examples/catalog/bin/sample_page.dart

```diff
@@ -22,22 +22,29 @@ class SampleError extends Error {
 // beginning of lines within the comment. A keyword's value is all of
 // the following text up to the next keyword or the end of the comment,
 // sans leading and trailing whitespace.
-const String sampleCatalogKeywords = r'^Title:|^Summary:|^Description:|^Classes:|^Sample:|^See also:';
+const String sampleCatalogKeywords =
+    r'^Title:|^Summary:|^Description:|^Classes:|^Sample:|^See also:';
```

Comment:
might be best to run it with a longer line length, this kind of change is definitely suboptimal